.gitignore

```diff
@@ -0,0 +1,119 @@
+# Logs
```

Comment:
This is adding quite a few files/directories that aren't 100% relevant to this project. Would you be able to add those to your global `.gitignore` instead?